### ISSUE TITLE
`ptx`: Do not use regex when -W options is not provided

### ISSUE THREAD

**Issue by allan-silva** [uutils/coreutils#3142]
`ptx` rely on regex even when -W option is not provided.
 https://github.com/uutils/coreutils/blob/65467ab317d885e14385aa0ff6bb55720160cc88/src/uu/ptx/src/ptx.rs#L118-L141

`ptx` should break words based on 8-bits charset when gnu extension is enabled or based on `[' ', '\t', '\n']` chars when gnu extension is disabled, but don't use regex when -W is not specified.

Ref:
[gnu ptx](https://github.com/coreutils/coreutils/blob/master/src/ptx.c#L469)

**Comment by allan-silva** [comment]
I realized this while work on #1763, I think it is better implement this first.

**Comment by mike-kfed** [comment]
Yes in the original C code that is correct, and probably done for performance reasons. However this can be solved by a regular expression too, and, even though I haven't written the original rust translation, I assume it was done with regex because it is easier to write. The project only strives to be compatible and not an exact copy. (also IMO it does not really matter how perfomant ptx is, it's virtually unused in the real world today.)

However I removed the reference in my PR to this issue, it is not for me to decide what goes into uutils. side-note: this discussion made me realise that I have a fat bug in my breakfile code that I will fix now, thanks :)
